### ISSUE TITLE
Streak Issue

### ISSUE THREAD

**Issue by Nadi-BrooklynCoder** [DenverCoder1/github-readme-streak-stats#674]
I just noticed my streak was not updating properly I was updating everyday and should have been past a 10 point streak.

**Comment by laxmariappan** [comment]
I had 3 contributions on April 30, as per GitHub contribution graph. But streak didn't include it. Not sure why!

https://github.com/laxmariappan

**Comment by DenverCoder1** [comment]
It all depends on timezones and I am not able to control which timezone the contributions are returned for

@laxmariappan For example, for me it also shows no contributions on April 30

![image](https://github.com/DenverCoder1/github-readme-streak-stats/assets/20955511/3de98d1a-b057-4f7b-8d4f-17def083585e)

Either the contributions are private or they were counted for a different day.

https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/docs/faq.md#why-doesnt-my-streak-stats-match-my-contribution-graph

**Comment by laxmariappan** [comment]
makes sense, thanks Jonah

**Comment by naveen-malla** [comment]
I made a commit in my private repo and it did not track. My streak was reset from 17 to 1 now. I even made that repo public and it still isn't updated.

**Comment by Bram1903** [comment]
Hello,

I am having the same issue. I am on a 448 day streak, but somehow on my profile it reset at January first. This must have changed somewhere last week. 

https://streak-stats.demolab.com/?user=bram1903
Shows 448 days ^^

![image](https://github.com/DenverCoder1/github-readme-streak-stats/assets/70259613/c92bc727-bb10-45d9-aaaf-7036086d5e86)
Shows 140 days on my profile page ^^

Kind regards, 
Bram

**Comment by shahtirth07** [comment]
Yeah I think the time-zone is an issue, around 10 pm PST everyday the git shows the day has changed to me and streaks never reflect.
